src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.c

```diff
@@ -43,36 +43,280 @@ const ActorInit Obj_Warp2block_InitVars = {
     (ActorFunc)ObjWarp2block_Update,
     (ActorFunc)ObjWarp2block_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Warp2block/func_80BA1DF0.s")
+typedef struct {
+    /* 0x0000 */ f32 scale;
+    /* 0x0004 */ f32 focus;
+    /* 0x0008 */ s16 params;
+} Warp2BlockSpawnData; // size = 0x000A
```

Comment:
since this is a small struct id recommend just having the numbers be a byte long `0x00`, but its not a huge deal. up to you if you want to change

Reply:
Yeah that makes sense, let me do that real quick.

Reply:
Done.